### PR TITLE
Update Raxml

### LINKED ISSUE #953: [Request An Update]: raxml
### What container needs an update?

RAxML (https://github.com/stamatak/standard-RAxML/releases/tag/v8.2.13) and RAxML-NG (https://github.com/amkozlov/raxml-ng/releases/tag/1.2.1) have new versions. 

The 8.2.12 Dockerfile is from 4 years ago.

This image needs 
- An updated base
- An app and test stage
- Updated labels
- CMD command
- Tests

This might also benefit from a builder stage.

### PR DESCRIPTION
**Pull request by Kincekara**
Closes #953
I separated Raxml and raxml-ng into different images as they should be. This PR should resolve all the issues mentioned in #953 

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: raxml/8.2.13/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+LABEL software="RAxML"
+LABEL software.version="${RAXML_VER}"
+LABEL description="A tool for Phylogenetic Analysis and Post-Analysis of Large Phylogenies"
+LABEL website="https://github.com/stamatak/standard-RAxML"
+LABEL license="https://github.com/stamatak/standard-RAxML/blob/v8.2.13/gpl-3.0.txt"
+LABEL maintainer="Kutluhan Incekara"
+LABEL maintainer.email="kutluhan.incekara@ct.gov"
+
+RUN apt-get update && apt-get install --no-install-recommends -y\
+    openmpi-bin
```
Comment (erinyoung) [inline raxml/8.2.13/Dockerfile:47]: Could you add autoclean to this?
```
apt-get autoclean && rm -rf /var/lib/apt/lists/*
```

**Comment by erinyoung** [comment]
The tests look good:

```

#9 1.647 (Chicken:1.794547,(((Cow:0.303295,(Seal:0.362885,Mouse:0.683504):0.189557):0.139758,Whale:0.243361):0.354493,Human:1.578070):0.698172,(Frog:0.558567,Loach:1.105978):0.187173);

```

```

#12 1.422 (Lemur:0.32537910426836053057,(Saimiri:0.43157643956775904037,((M_sylvanus:0.07088921537201800516,(M_fascicul:0.05523388801310591401,(Macaca:0.01608115262138561444,M_mulatta:0.02239621504004839225):0.03388122119428586820):0.04268050828482932574):0.23505406685566176961,(Gibbon:0.16511638061802774713,((Gorilla:0.05879059075376137644,(Chimp:0.05900565905273858630,H_sapiens:0.04840728794905584109):0.02375800215551397351):0.07400567918350323127,Orangutan:0.13273893510289544451):0.04406182262600038396):0.10886147176113297741):0.10360382224976649734):0.24821952513486267300,Tarsius:0.47478261882585243159):0.0;

```

**Comment by erinyoung** [comment]
This looks great! It also looks like you put a lot of work into it.

File: raxml/8.2.13/Dockerfile (hunk 1 above)
Comment (Kincekara) [reply raxml/8.2.13/Dockerfile:47]: good catch!

**Comment by Kincekara** [comment]
@erinyoung I made the changes. I think this is ready.

**Review by erinyoung — APPROVED** [review]
Looks good. I'll get these two deployed.

**Comment by erinyoung** [comment]
Thank you for putting these together!

The two deployments can be found at 
https://github.com/StaPH-B/docker-builds/actions/runs/9502319996
https://github.com/StaPH-B/docker-builds/actions/runs/9502326488

**Pull request merged**